src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -2079,6 +2086,92 @@ impl Coordinator {
         };
     }
 
+    async fn render_explain_pushdown<'a>(
+        &mut self,
+        ctx: &ExecuteContext,
+        mz_now: ResultSpec<'a>,
+        import_as_of: Antichain<Timestamp>,
+        imports: impl IntoIterator<Item = (GlobalId, MapFilterProject)>,
+    ) -> Result<ExecuteResponse, AdapterError> {
+        use futures::TryStreamExt;
+
+        if import_as_of.is_empty() {
+            Err(AdapterError::Internal(
+                "cannot explain stats as-of the empty antichain".to_string(),
+            ))?;
+        }
```

Comment:
I've added this clause because the added SLTs showed that we were trying to fetch stats for inputs that had advanced to `[]`, which can happen with constant MVs and similar.

Right now I've made this an error, but it would also be possible to make this a success with zero rows... under the theory that in practice this will render no sources at all next time the cluster rehydrates. Either seems fine to me, but I'm happy to swap this if any readers have a preference.

Reply:
I'd say erroring is ok: If a user is calling EXPLAIN FILTER PUSHDOWN on an MV that won't create a dataflow on the next restart (either because it's a constant, or it's past its last refresh, etc.), that feels like some kind of mistake, so it's good if it's called out.

But maybe make it a non-internal error. I think `AdapterError::Internal` is reserved for situations where there is some bug in the system.